<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>673</y>
     <width>1056</width>
     <height>77</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox { 
	background-color: rgb(250, 250, 250);
 border: none;
 border-top: 1px solid gray; 
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="widget5" native="true">
    <property name="geometry">
     <rect>
      <x>384</x>
      <y>7</y>
      <width>291</width>
      <height>43</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_1">
     <item>
      <widget class="QPushButton" name="pushButton_state_change">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background-repeat:no-repeat;
background-position:center center;
border:none;
color:white;
}
QPushButton:hover{
background-repeat:no-repeat;
background-position:center center;
}
QPushButton:pressed{
background-repeat:no-repeat;
background-position:center center;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/loop.png</normaloff>:/images/loop.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_PreviousSong_4">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background-repeat:no-repeat;
background-position:center center;
border:none;
color:white;
}
QPushButton:hover{
background-repeat:no-repeat;
background-position:center center;
}
QPushButton:pressed{
background-repeat:no-repeat;
background-position:center center;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/play-previous.png</normaloff>:/images/play-previous.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_PlaySong_4">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background-repeat:no-repeat;
background-position:center center;
border:none;
color:white;
}
QPushButton:hover{
background-repeat:no-repeat;
background-position:center center;
}
QPushButton:pressed{
background-repeat:no-repeat;
background-position:center center;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/play.png</normaloff>:/images/play.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_PlayNextSong_4">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background-repeat:no-repeat;
background-position:center center;
border:none;
color:white;
}
QPushButton:hover{
background-repeat:no-repeat;
background-position:center center;
}
QPushButton:pressed{
background-repeat:no-repeat;
background-position:center center;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/play-next.png</normaloff>:/images/play-next.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_lyrics">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background-repeat:no-repeat;
background-position:center center;
border:none;
color:white;
}
QPushButton:hover{
background-repeat:no-repeat;
background-position:center center;
}
QPushButton:pressed{
background-repeat:no-repeat;
background-position:center center;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/lyrics.png</normaloff>:/images/lyrics.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
    <zorder>pushButton_PlayNextSong_4</zorder>
    <zorder>pushButton_PreviousSong_4</zorder>
    <zorder>pushButton_state_change</zorder>
    <zorder>pushButton_PlaySong_4</zorder>
    <zorder>pushButton_lyrics</zorder>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_PlayTime">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>48</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_PlayProgress">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>53</y>
      <width>391</width>
      <height>8</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_DisplaySongTitle">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:16px;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="record">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>11</y>
      <width>56</width>
      <height>56</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	border-radius:28
;
	
	
	background-color: rgb(0, 0, 0);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget4" native="true">
    <property name="geometry">
     <rect>
      <x>846</x>
      <y>20</y>
      <width>181</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_download">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background-repeat:no-repeat;
background-position:center center;
border:none;
color:white;
}
QPushButton:hover{
background-repeat:no-repeat;
background-position:center center;
}
QPushButton:pressed{
background-repeat:no-repeat;
background-position:center center;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/downLoad.png</normaloff>:/images/downLoad.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_playlist">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background-repeat:no-repeat;
background-position:center center;
border:none;
color:white;
}
QPushButton:hover{
background-repeat:no-repeat;
background-position:center center;
}
QPushButton:pressed{
background-repeat:no-repeat;
background-position:center center;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/playlist.png</normaloff>:/images/playlist.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_SoundYesNo">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background-repeat:no-repeat;
background-position:center center;
border:none;
color:white;
}
QPushButton:hover{
background-repeat:no-repeat;
background-position:center center;
}
QPushButton:pressed{
background-repeat:no-repeat;
background-position:center center;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/sound-on.png</normaloff>:/images/sound-on.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider_Volume">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background-repeat:no-repeat;
background-position:center center;
border:none;
color:white;
}
QPushButton:hover{
background-repeat:no-repeat;
background-position:center center;
}
QPushButton:pressed{
background-repeat:no-repeat;
background-position:center center;
}
</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>36</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="widget1" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1056</width>
     <height>63</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(234, 62, 62);</string>
   </property>
   <widget class="QLabel" name="app_name">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>151</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>华文琥珀</family>
      <pointsize>22</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(245, 245, 245);</string>
    </property>
    <property name="text">
     <string>汽泡音乐</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget2" native="true">
    <property name="geometry">
     <rect>
      <x>933</x>
      <y>15</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton_Min">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background-repeat:no-repeat;
background-position:center center;
border:none;
color:white;
}
QPushButton:hover{
background-repeat:no-repeat;
background-position:center center;
}
QPushButton:pressed{
background-repeat:no-repeat;
background-position:center center;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/minsize.png</normaloff>:/images/minsize.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_About">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background-repeat:no-repeat;
background-position:center center;
border:none;
}
QPushButton:hover{
background-repeat:no-repeat;
background-position:center center;
}
QPushButton:pressed{
background-repeat:no-repeat;
background-position:center center;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/about.png</normaloff>:/images/about.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Exit">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background-repeat:no-repeat;
background-position:center center;
border:none;
color:white;
}
QPushButton:hover{
background-repeat:no-repeat;
background-position:center center;
}
QPushButton:pressed{
background-repeat:no-repeat;
background-position:center center;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/close.png</normaloff>:/images/close.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
    <zorder>pushButton_About</zorder>
    <zorder>pushButton_Min</zorder>
    <zorder>pushButton_Exit</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>206</x>
      <y>10</y>
      <width>691</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_SearchSong">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>10</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:14px;
background-color: rgba(255, 255, 255,200);</string>
     </property>
     <property name="text">
      <string>Search...</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/images/search.png</normaloff>:/images/search.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_St">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:14px;

background-color: rgba(255, 255, 255,200);</string>
     </property>
     <property name="text">
      <string>动手搜一搜</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/new/prefix1/ResImages/MDriect.png</normaloff>:/new/prefix1/ResImages/MDriect.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_InputSongs">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:14px;

background-color: rgba(255, 255, 255,200);</string>
     </property>
     <property name="text">
      <string>无形之物</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="icon">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-repeat:no-repeat;
background-position:center center;
border:none;
color:white;
}
QPushButton:hover{
background-repeat:no-repeat;
background-position:center center;
}
QPushButton:pressed{
background-repeat:no-repeat;
background-position:center center;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/images/icon.png</normaloff>
      <disabledoff>:/images/icon.png</disabledoff>
      <disabledon>:/images/icon.png</disabledon>:/images/icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget3" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>64</y>
     <width>1061</width>
     <height>611</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox
{
font-size:14px;
color:black
}</string>
      </property>
      <property name="title">
       <string>音乐搜索列表</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>16</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QListWidget" name="listWidget_searchList">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(247, 249, 252);
font-size:18px;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="search_page">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>513</width>
          <height>593</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox
{
font-size:14px;
color:black
}</string>
        </property>
        <property name="title">
         <string>音乐播放列表</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>16</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QTextEdit" name="plainTextEdit_SongList">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size:20px;

background-color: rgba(247, 249, 252);</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="lyrics_page">
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>513</width>
          <height>593</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox
{
font-size:14px;
color:black
}</string>
        </property>
        <property name="title">
         <string>歌词</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>16</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QTextEdit" name="plainTextEdit_lyrics">
           <property name="styleSheet">
            <string notr="true">font-size:20px;

background-color: rgba(247, 249, 252);</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>widget1</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>widget3</zorder>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
